EventFilter/SiStripRawToDigi/interface/SiStripFEDBufferComponents.h

```diff
@@ -40,22 +40,49 @@ namespace sistrip {
   enum FEDReadoutMode { READOUT_MODE_INVALID=INVALID,
                         READOUT_MODE_SCOPE=0x1,
                         READOUT_MODE_VIRGIN_RAW=0x2,
+                        READOUT_MODE_ZERO_SUPPRESSED_LITE10=0x3,
+                        READOUT_MODE_ZERO_SUPPRESSED_LITE10_CMOVERRIDE=0x4,
+                        READOUT_MODE_ZERO_SUPPRESSED_LITE8_TOPBOT=0x5,
                         READOUT_MODE_PROC_RAW=0x6,
+                        READOUT_MODE_ZERO_SUPPRESSED_LITE8_TOPBOT_CMOVERRIDE=0x7,
+                        READOUT_MODE_ZERO_SUPPRESSED_LITE8_CMOVERRIDE=0x8,
+                        READOUT_MODE_ZERO_SUPPRESSED_LITE8_BOTBOT=0x9,
                         READOUT_MODE_ZERO_SUPPRESSED=0xA,
-                        READOUT_MODE_ZERO_SUPPRESSED_LITE=0xC,
+                        READOUT_MODE_ZERO_SUPPRESSED_FAKE=0xB,
```

Comment:
There seem to be a mismatch here between the enum value 11 -> fake mode, 13=LITE8_BOTBOT_CMO, and the method  DataFormats/SiStripCommon/interface/ConstantsForHardwareSystems.h line 143 where LITE8_BB_CMO is = 11 and 13 is not given to anything....